test/WorkflowCore.TestAssets/Utils.cs

```diff
@@ -32,6 +32,11 @@ public static string GetTestDefinitionDynamicJson()
         {
             return File.ReadAllText("stored-dynamic-definition.json");
         }
+
+        public static string GetTestDefinitionJsonInputExc()
+        {
+            return File.ReadAllText("stored-definition-input-exc.json");
```

Comment:
What does `-exc` stand for?  Could we make it more explicit?

Reply:
Ok.  I changed the file name and appropriate methods/files.